geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/Utils.scala

```diff
@@ -61,8 +61,8 @@ object Utils {
   object Formats {
     val CSV     = "csv"
     val TSV     = "tsv"
-    val TIFF    = "tiff"
-    val DTED    = "dted"
+    val TIFF    = "GTiff"
```

Comment:
I have a strong preference to spell out 'geotiff' unless there is a strong precedence for the abbreviation GTiff.